### DISCUSSION TITLE
Improve RTL spacing classes

### DISCUSSION THREAD

**Discussion by KareemDa** [tailwindlabs/tailwindcss#9414] [Ideas]
My idea is to use:
- **"Start -> s"**  instead of **"Left-> l"**
- **"End-> e"**  instead of **"Right-> r"**

and internally, tailwind will detect the direction and add "right" or "left" class 

this can be applied to:
- margin
- border
- padding
- text-align

**Comment by KareemDa** [comment]
Using this guideline, if we're building a multi-directional website and want to add margin before text:

we can use: `ms-2` instead of `ltr:ml-2 rtl:mr-2`

**Comment by thibaudcolas** [comment]
See #1483 which discusses this extensively. Logical properties and values are a concept in CSS itself, no need for Tailwind itself to reinvent this.
